about.css

```diff
@@ -0,0 +1,325 @@
+.contnt {
+  display: flex;
+  justify-content: flex-end;
+  padding: 20px;
+  align-items: flex-end;
+}
+
+.banner2 {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-image: url(./images/background.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
```

Comment:
- [ ] I think it would be better to add a [universal selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors) ```*``` in your CSS file to match elements of any type and set rules to avoid repetition in your codes, 👍 
   - Currently, you have used ```font-family: 'Lato', sans-serif;``` **10** times which is not a good practice, this can be grabbed in the universal selector  😎
   - Your may also set the default ```margin and padding to 0``` to remove the default added spacing between our elements on the page 👍 

### Example
```
* {
   margin: 0;
   padding: 0;
   font-family: 'Lato', sans-serif;
}
```

---

This space around your page will be removed✨😉
![image](https://user-images.githubusercontent.com/102757126/217872393-25afaad5-f858-43a9-a275-314950932cc4.png)

Reply:
Addressed.